dzVents/runtime/tests/testUtils.lua

```diff
@@ -80,8 +80,12 @@ describe('event helpers', function()
 		assert.is_true(utils.fileExists('testfile'))
 	end)
 
-	it('should return true for os.execute', function()
-		assert.is_true(utils.osExecute('dir'))
+	it('should return true for os.execute (echo)', function()
+		assert.is_nil(utils.osExecute('echo test > testfile.out'))
```

Comment:
You compare it with a nill value, but a line above stays "should return true for os.execute (echo)".
Must be nill too? Or compare it with is_true?

Also on line 87 ;)

Reply:
Thx and good catch. Will correct in next version. Luckily this is only a message in a test-set.